### PR TITLE
Add telemetry support

### PR DESCRIPTION
**Pull request by rchiodo**
Also move datascience ui to root branch in prep of bundling

### CODE REVIEW

File: src/client/datascience/editor-integration/codewatcher.ts [hunk 1]
```diff
@@ -1,12 +1,14 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 'use strict';
-import { CodeLens, Command, Position, Range, Selection, TextDocument, TextEditorRevealType, window} from 'vscode';
+import { CodeLens, Command, Position, Range, Selection, TextDocument, TextEditorRevealType, window } from 'vscode';
```
Comment (IanMatthewHuff) [inline src/client/datascience/editor-integration/codewatcher.ts]: Is the extra empty line here intentional?

Comment (rchiodo) [reply src/client/datascience/editor-integration/codewatcher.ts]: The TypeScript Hero reformats stuff that way. So I just left it. 

---
In reply to: [229386813](https://github.com/vscode-python-datascience/vscode-python/pull/47#discussion_r229386813) [](ancestors = 229386813)

File: src/datascience-ui/history-react/cell.css [hunk 2]
```diff
@@ -46,6 +46,8 @@
 .cell-output {
     margin-top: 5px;
     background: var(--vscode-notifications-background);
+    white-space: pre-wrap;
```
Comment (rchiodo) [inline src/datascience-ui/history-react/cell.css]: This fixes Rong's convolution matrix

File: src/test/datascience/history.unit.test.tsx [hunk 3]
```diff
@@ -28,18 +27,24 @@ import { mount } from 'enzyme';
...
 import { IDisposableRegistry, ILogger } from '../../client/common/types';
-import { Cell } from '../../client/datascience/history-react/cell';
-import { MainPanel } from '../../client/datascience/history-react/mainPanel';
 import { HistoryProvider } from '../../client/datascience/historyProvider';
 import { JupyterServerProvider } from '../../client/datascience/jupyterServerProvider';
-import { IVsCodeApi } from '../../client/datascience/react-common/postOffice';
-import { IHistoryProvider, IJupyterServerProvider  } from '../../client/datascience/types';
+import { IHistoryProvider, IJupyterServerProvider } from '../../client/datascience/types';
 import { IServiceContainer } from '../../client/ioc/types';
+import { IVsCodeApi } from '../../datascience-ui//react-common/postOffice';
```
Comment (IanMatthewHuff) [inline src/test/datascience/history.unit.test.tsx]: extra / also Typescript Hero?

File: src/datascience-ui/history-react/cell.css (hunk 2 above)
Comment (rchiodo) [reply src/datascience-ui/history-react/cell.css]: Well and the font monospace. That's what Jupyter uses in its css

---
In reply to: [229387579](https://github.com/vscode-python-datascience/vscode-python/pull/47#discussion_r229387579) [](ancestors = 229387579)

File: src/test/datascience/history.unit.test.tsx (hunk 3 above)
Comment (rchiodo) [reply src/test/datascience/history.unit.test.tsx]: Yeah. I'm not manually editing these anymore. 

---
In reply to: [229387630](https://github.com/vscode-python-datascience/vscode-python/pull/47#discussion_r229387630) [](ancestors = 229387630)

**Comment by rchiodo** [comment]
I wanted to write a unit test to verify all of our telemetry goes through, but that would require some changes to the telemetry infrastructure. I was going to discuss that with Don and do a follow up (or make it part of our test writing we're starting on)

**Pull request merged**